### PR TITLE
TDS-182 create regional and district panel

### PR DESCRIPTION
**Pull request by esoliani**
TSD-185 add a reset view in this new panel

### CODE REVIEW

File: app/views/poligon-number.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="widget col-md-12">
+  <span class="widgettitle">{{name}}</span>
+</div>
+<p class="widgetnumber">
+  <span>
+    <img class="poligonimage" ng-src="images/{{image}}">
+    <span ng-show="(value|exists) && value !== NaN">{{ value }}</span>
+    <span ng-hide="(value|exists) && value !== NaN" class="medium-character missing-data" translate="chart.metric.missing-data" style="position: relative;">NA</span >
```
Comment (uniphil) [inline app/views/poligon-number.html]: Do we need the `position` css rule? If so, maybe we can move it to one of the stylesheets?

File: app/scripts/controllers/dashboard.coffee [hunk 2]
```diff
@@ -467,6 +467,9 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', (
         # side-effects: de-select any selected school, zoom out to see everything
         resetView = ->
           $scope.selectSchool null
+          $scope.setPolyType null
+          $scope.selectPoly null
+          $scope.setViewMode 'schools'
```
Comment (uniphil) [inline app/scripts/controllers/dashboard.coffee:472]: ok

File: app/views/dashboard.html [hunk 3]
```diff
@@ -195,7 +196,7 @@ <h4 translate="legend.ptratio">Pupil-Teacher Ratio</h4>
               </div>
             </div>
           </div>
-          <div class="container-fluid perfomance-gauges" ng-hide="selectedSchool|exists">
+          <div class="container-fluid perfomance-gauges" ng-hide="(selectedSchoolCode|exists) || (selectedPolyId|exists)">
```
Comment (uniphil) [inline app/views/dashboard.html:199]: The logic in the html is getting kind of complicated, but oh well it works and we're almost done, and I can't think of anything obviously better. :shipit:

Comment (esoliani) [inline app/views/dashboard.html:199]: I could move this logic to a scope function

File: app/views/poligon-number.html (hunk 1 above)
Comment (esoliani) [inline app/views/poligon-number.html]: fixed

File: app/views/dashboard.html (hunk 3 above)
Comment (uniphil) [inline app/views/dashboard.html:199]: That's a good idea... how about: if we have any new bug come in related to sidebar stuff visibility, we will move logic for that into a scope function. For now lets leave it.

**Pull request merged**